admin/Caddyfile

```diff
@@ -0,0 +1,49 @@
+# Caddy's configuration file to serve up frontend. (Require Caddy installed/downloaded in system)
+# Local: Assume local build is ready at './dist/admin'; at root working directory run command "caddy run --watch" at console.
+#        Then access local frontend app: http://localhost:4300/admin/
+
+:4200 {
+    log {
+        level DEBUG
+        format json
+        output stdout
+    }
+
+    # Good practice to compress responses.
+    encode zstd gzip
+
+    root /* ./dist/admin
+
+    header {
+        # allow caching for 5 minutes, for refreshing after deployments.
+        Cache-Control "public, max-age=300"
+        Content-Security-Policy "connect-src 'self' https://*.gov.bc.ca http://*.gov.bc.ca;form-action 'self';img-src 'self' https://server.arcgisonline.com data: https://services.arcgisonline.com data: https://maps.gov.bc.ca;frame-src 'self' https://*.gov.bc.ca;default-src 'self';base-uri 'self';block-all-mixed-content;font-src 'self' https: data:;frame-ancestors 'self';object-src 'none';script-src 'self';style-src 'self' https: 'unsafe-inline';upgrade-insecure-requests;"
```

Comment:
You could add a comment explaining that the header definition has to remain on a single line.